app/src/forms/admin/controller.js

```diff
@@ -87,6 +87,34 @@ module.exports = {
     }
   },
 
+  //
+  // APIs
+  //
```

Comment:
The backend test coverage is looking great, but it looks like the additions to this file got missed.